spec/requests/v1/pacbio/runs/plates_spec.rb

```diff
@@ -3,6 +3,9 @@
 require 'rails_helper'
 
 RSpec.describe 'PlatesController', type: :request do
+  let!(:version10) { create(:pacbio_smrt_link_version, name: 'v10', default: true) }
+  let!(:version11) { create(:pacbio_smrt_link_version, name: 'v11') }
+
```

Comment:
Same again :)

Reply:
I moved the default one into a before block and removed the other one. I also removed the symbol as there is no direct access to the version.